build/charts/antrea-windows/conf/ovs/Install-OVSDriver.ps1

```diff
@@ -1,37 +1,14 @@
 $ErrorActionPreference = "Stop"
 $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
 $mountPath = ($mountPath.Replace('\', '/')).TrimEnd('/')
-$OVSDriverDir = "$mountPath\openvswitch\driver"
-
-# Check if OVSExt driver is already installed
-$driverStatus = netcfg -q ovsext
-if ($driverStatus -like '*not installed*') {
-  # Install OVS Driver
-  $result = netcfg -l $OVSDriverDir/ovsext.inf -c s -i OVSExt
-  if ($result -like '*failed*') {
-    Write-Host "Failed to install OVSExt driver: $result"
-    exit 1
-  }
-  Write-Host "OVSExt driver has been installed"
+$OVSInstallScript = "$mountPath\k\antrea\Install-OVS.ps1"
+if (-not (Test-Path $OVSInstallScript)) {
+  Write-Host "Installation script not found: $OVSInstallScript, you may be using an invalid antrea-windows container image"
+  exit 1
 }
-
-# Check if the VC redistributable is already installed.
-$OVSRedistDir="$mountPath\openvswitch\redist"
-if (Test-Path $OVSRedistDir) {
-  $dllFound = $false
-  $paths = $env:PATH -split ';'
-  foreach ($path in $paths) {
-    $dllFiles = Get-ChildItem -Path $path -Filter "vcruntime*.dll" -File -ErrorAction SilentlyContinue
-    if ($dllFiles.Count -gt 0) {
-      $dllFound = $true
-      break
-    }
-  }
-
-  # vcruntime dlls are not installed on the host, then install the binaries.
-  if (-not $dllFound) {
-    Get-ChildItem $OVSRedistDir -Filter *.exe | ForEach-Object {
-      Start-Process -FilePath $_.FullName -Args '/install /passive /norestart' -Verb RunAs -Wait
-    }
-  }
+& $OVSInstallScript -LocalFile "$mountPath/openvswitch" -InstallUserspace $false
+If (!$?) {
+  Write-Host "Failed to install OVS"
```

Comment:
We could capture error log here with exact reason of failure during ovs installation ?
  e.g Write-Host "Error Message: $errorMessage"

Reply:
I don't think it is necessary to print the errorMessage again, the installation details are printed when running the install script (as we use a Log function to output to both log file and stdout).

Reply:
@wenyingd Given that OVS is running in the container, is it more accurate to say “Failed to install OVS driver”?